thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -313,3 +315,11 @@ func (r *EvalFnRunner) postRun(_ *cobra.Command, args []string) {
 	}
 	pkgutil.FormatPackage(path)
 }
+
+// addDefaultImagePathPrefix adds default gcr.io/kpt-fn/ path prefix to image if only image name is specified
+func (r *EvalFnRunner) addDefaultImagePathPrefix() {
+	if r.Image != "" && !strings.Contains(r.Image, "/") &&
+		r.ImagePullPolicy == string(fnruntime.AlwaysPull) {
```

Comment:
Why is this dependent on image pull policy? It's orthogonal concern.

Reply:
Moved this method to fnruntime